erica_snider/assignments/JavaScript/jQueryFunctions/jQueryFunctions.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+  <script type="text/javascript">
+  "use strict";
+  $( document ).ready(function() {
+    var temp;
+    $("#click button").click(function(){
+      $("#click button").css("background-color", "#ffaf8c");
+    })
+    $("#hide button").click(function(){
+      $("#hide p").hide();
+      $("#show p").hide();
+    })
+    $("#show button").click(function(){
+      $("#show p").show();
+      $("#hide p").show();
+    })
+    $("#toggle button").click(function(){
+      $("#toggle p").toggle("slow", function(){
+        $("#toggle p").html("Toggle'd!");
+      });
+    })
+    $("#slideUp button").click(function(){
+      $("#slideDown p").slideUp();
+      $("#slideUp p").slideUp();
+    })
+    $("#slideDown button").click(function(){
+      $("#slideUp p").slideDown();
+      $("#slideDown p").slideDown();
+    })
+    $("#slideToggle button").click(function(){
+      $("#slideToggle p").slideToggle("slow", function(){
+        $("#slideToggle p").html("slideToggle'd!");
+      });
+    })
+    $("#fadeOut button").click(function(){
+      $("#fadeOut p").fadeOut(1000);
+      $("#fadeIn p").fadeOut(1000);
+    })
+    $("#fadeIn button").click(function(){
+      $("#fadeIn p").fadeIn(1000);
+      $("#fadeOut p").fadeIn(1000);
+    })
+    $("#addClass button").click(function(){
+      $("#addClass p").addClass("bold-n-blue");
+    })
+    $("#before button").click(function(){
+      $("#before p").before("<p>I'm a new paragraph!</p>"); // issue: adds 1, then 2, then 4, ...
+    })
+    $("#after button").click(function(){
+      $("#after p").after("<p>I'm a new paragraph!</p>"); // issue: adds 1, then 2, then 4, ...
```

Comment:
Sort of an issue.  When you first click the button, there's only 1 paragraph so it adds 1.  But the next time you click it you now have 2 paragraphs inside the same div, each with the same selector, so your jQuery now adds another paragraph after each one that exists, so it doubles every time.   One option to fix that would be to give your initial existing paragraph a unique id and select it that way.